README.md

```diff
@@ -1,31 +1,45 @@
-# TODO-MVP-ContentProviders
+# todo-mvp-contentproviders
 
-It is based on the [TODO-MVP-Loaders](https://github.com/googlesamples/android-architecture/tree/todo-mvp-loaders) sample and uses a Content Provider to retrieve data into the repository.
+This version of the app is called todo-mvp-contentproviders. It is based on the [todo-mvp-loaders](https://github.com/googlesamples/android-architecture/tree/todo-mvp-loaders) sample, and adds a [content provider]((https://developer.android.com/guide/topics/providers/content-providers.html)) to manage access to the underlying SQLite database storage. Content providers provide a standard interface for allowing access to data in one process from code running in a different process. This sample demonstrates how to add a content provider to the [todo-mvp-loaders](https://github.com/googlesamples/android-architecture/tree/todo-mvp-loaders) sample. 
 
-<img src="https://github.com/googlesamples/android-architecture/wiki/images/mvp-contentproviders.png" alt="Diagram"/>
+You could then use content providers to support additional features that are not covered by this sample, providing the following possible benefits:
 
-The advantages of Content Providers, from the [Content Provider documentation page](http://developer.android.com/guide/topics/providers/content-providers.html), are:
+ * Allow you to securely share data stored in your application with other applications.
+ * Add support for custom searches in your application.
+ * Develop widgets to access data in your application.
```

Comment:
I don't see the distinction between 'to' and 'that' in this case but I've updated to read:

Develop widgets which access data in your app.